### PR TITLE
[v7r2]dirac-install: bashrc does not contain GFAL or ARC path when installe…

### PR DESCRIPTION
**Pull request by chaen**
…d with diracos

BEGINRELEASENOTES

*Core
CHANGE: dirac-install with diracos: remove TERMINFO, RRD_DEFAULT_FONT, GFAL and ARC paths from bashrc
FIX: Late import of MonitoringReport

ENDRELEASENOTES

### CODE REVIEW

**Comment by chrisburr** [comment]
`TERMINFO` and `RRD_DEFAULT_FONT` should also be moved. Though the value of `RRD_DEFAULT_FONT` doesn't exist in DIRACOS 1.

**Comment by chaen** [comment]
For TERMINFO we could set it in `DIRACOS`, and in `dirac-install` set it to `TERMINFO=$TERMINFO:/usr/share/terminfo:/etc/terminfo`

FOR `RRD_DEFAULT_FONT` I guess we can chose any of the font in `diracos/usr/share/fonts/dejavu/`

**Comment by chaen** [comment]
Fixes https://github.com/DIRACGrid/DIRAC/issues/4793

**Comment by chaen** [comment]
@chrisburr any thought about the last comment for `TERMINFO` and `RRD_DEFAULT_FONT` ?

**Comment by chrisburr** [comment]
* `TERMINFO`: Does it need to be set at all? If it does then it should be set in the `diracosrc` as it has nothing to do with DIRAC itself.
* `RRD_DEFAULT_FONT`: As it's been set to a file that doesn't exist for who-knows-how-long I'd guess we can just remove it?

File: WorkloadManagementSystem/Service/MatcherHandler.py [hunk 1]
```diff
@@ -118,7 +119,9 @@ def export_getActiveTaskQueues():
 
 ##############################################################################
   types_getMatchingTaskQueues = [dict]
+  # int keys are cast into str
 
+  @ignoreEncodeWarning
```
Comment (fstagni) [inline WorkloadManagementSystem/Service/MatcherHandler.py:124]: Apply also to `export_getActiveTaskQueues` (also for client, I think)

**Comment by chaen** [comment]
According to @phicharp `TERMINFO` was needed for `emacs` to be used on the VOBOX but I think it can be removed, unless @andresailer really needs it. 
Yes, I'll get ride of `RDD_*`

File: WorkloadManagementSystem/Service/MatcherHandler.py (hunk 1 above)
Comment (chaen) [reply WorkloadManagementSystem/Service/MatcherHandler.py:124]: OK

File: Core/scripts/dirac-install.py [hunk 2]
```diff
@@ -2476,11 +2462,6 @@ def createBashrcForDiracOS():
       lines.extend(['# new OpenSSL version require OPENSSL_CONF to point to some accessible location',
                     'export OPENSSL_CONF=/tmp'])
 
-      # gfal2 requires some environment variables to be set
-      # Note: eventually this line should disappear as already set by diracosrc
-      lines.extend(['# Gfal2 configuration and plugins',
-                    'export GFAL_CONFIG_DIR=$DIRACOS/etc/gfal2.d',
-                    'export  GFAL_PLUGIN_DIR=$DIRACOS/usr/lib64/gfal2-plugins/'])
```
Comment (fstagni) [inline Core/scripts/dirac-install.py:2483]: Suppose that this version of dirac-install will be used for installing DIRAC v6r21. What would happen?

Comment (chaen) [reply Core/scripts/dirac-install.py:2483]: this is only for `diracos` so no risk I would say

**Comment by andresailer** [comment]
If TERMINFO is needed to run emacs within diracos, then yes I need it!

**Comment by chaen** [comment]
well, try and let me know if it works without :-D

**Comment by fstagni** [comment]
I took the liberty of marking this "ready for review" and at the same time update the release notes.

**Review by atsareg — APPROVED** [review]

**Pull request merged**